broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -32,11 +38,79 @@ const CloudControllerError = {
   }
 };
 
+
 class ServiceFabrikApiController extends FabrikBaseController {
   constructor() {
     super();
   }
 
+  /**
+   * @param opts
+   * @param opts.resourceId
+   * @param opts.annotationId
+   * @param opts.start_state
+   * @param opts.started_at
+   */
+  static getResourceAnnotationStatus(opts) {
+    logger.info(`Waiting ${CONST.EVENTMESH_POLLER_DELAY} ms to get the annotation state`);
+    return Promise.delay(CONST.EVENTMESH_POLLER_DELAY)
+      .then(() => eventmesh.server.getAnnotationState({
+        resourceId: opts.resourceId,
+        annotationName: CONST.OPERATION_TYPE.BACKUP,
+        annotationType: CONST.RESOURCE_NAMES.DEFAULT_BACKUP,
+        annotationId: opts.annotationId
+      })).then(state => {
+        const duration = (new Date() - opts.started_at) / 1000;
+        const backup_start_timeout = CONST.BACKUP.BACKUP_START_TIMEOUT; //sec
+        logger.info(`Polling for ${opts.start_state} duration: ${duration} `);
+        if (duration > backup_start_timeout) {
+          throw new Timeout(`Backup not picked up from queue`);
```

Comment:
Put one identifier in the logger.

Reply:
Changed it to `Backup with guid ${opts.operationId} not picked up from the queue`